client/apps/edit/components/admin/test/index.coffee

```diff
@@ -1,131 +1,87 @@
-      @view.toggleScheduled(@e)
-      @view.article.get('scheduled_publish_at').should.containEql moment(dateAndTime, 'MM/DD/YYYY HH:mm').toDate()
-      delete global.confirm
-
-    it 'can unset the scheduled publish time', ->
-      @view.article.set {scheduled_publish_at: '2015-11-11T22:59:00.000Z'}
-      @view.toggleScheduled(@e)
-      should.equal(@view.article.get('scheduled_publish_at'), null)
-
-  describe '#setupPublishDate', ->
-
-    it 'can set the publish date and time', ->
-      date = '11/11/11'
-      time = '12:30'
-      dateAndTime = date + ' ' + time
-      @view.$('.edit-admin-input-date').val(date)
-      @view.$('.edit-admin-input-time').val(time)
-      @view.$('.edit-admin-input-time').trigger('blur')
-      @view.article.get('published_at').should.containEql moment(dateAndTime).toISOString()
 
-    it 'validates the date before trying to set it', ->
-      currentDate = moment().format('L')
-      date = '13/40/1991'
-      time = '1:0:00'
-      dateAndTime = date + ' ' + time
-      @view.$('.edit-admin-input-date').val(date)
-      @view.$('.edit-admin-input-time').val(time)
-      @view.$('.edit-admin-input-time').trigger('blur')
-      moment(@view.article.get('published_at')).format('L').should.equal currentDate
+  it '#componentWillMount fetches featured and mentioned artists/artworks', ->
+    @component.props.article.fetchFeatured = sinon.stub()
+    @component.props.article.fetchMentioned = sinon.stub()
+    @component.componentWillMount()
+    @component.props.article.fetchFeatured.called.should.be.ok
+    @component.props.article.fetchMentioned.called.should.be.ok
+
+  it 'Adds a section to activeSections on click', ->
+    r.simulate.click r.find @component, 'featuring'
+    $(ReactDOM.findDOMNode(@component)).find('.edit-admin--featuring').hasClass('active').should.eql true
+    @component.state.activeSections.should.eql ['article', 'verticals-tags', 'featuring']
+
+  it 'Removes a section from activeSections on click', ->
+    r.simulate.click r.find @component, 'verticals-tags'
+    $(ReactDOM.findDOMNode(@component)).find('.edit-admin--verticals-tags').hasClass('active').should.eql false
+    @component.state.activeSections.should.eql ['article']
+
+  it '#getActiveSection returns active class if section is active', ->
+    @component.getActiveSection('article').should.eql ' active'
+    @component.getActiveSection('featuring').should.eql ''
+
+  it '#isActiveSection returns true if section is active', ->
+    @component.isActiveSection('article').should.eql true
+    @component.isActiveSection('featuring').should.eql false
+
+  it '#onChange updates the article attributes', ->
+    @component.onChange('tier', 2)
+    @component.props.article.get('tier').should.eql 2
+
+  it 'If unpublished, #onChange does not save and shows indicator', ->
+    @component.onChange('tier', 2)
+    @component.props.article.save.callCount.should.eql 0
+    # is there a way to show jquery changes outside the component?
```

Comment:
Just in the test context -- wanted to prove the header got a class, but it isn't rendered here.